### ISSUE TITLE
Persona A11y Pass.

### ISSUE THREAD

**Issue by TestAccessibilityGIT** [OfficeDev/office-ui-fabric-react#7643]
Tracking all of the issues brought up with the 1/11 accessibility pass on Persona control 
Details on individual bugs can be found on[ SharePoint](https://microsoft-my.sharepoint.com/:x:/r/personal/v-amaver_microsoft_com/_layouts/15/Doc.aspx?sourcedoc=%7B49e2b9a0-7b43-492b-8dba-1f1170e59b19%7D&action=default&uid=%7B49E2B9A0-7B43-492B-8DBA-1F1170E59B19%7D&ListItemId=1663&ListId=%7B395E1ECB-5002-440E-81C5-6A090E98077C%7D&odsp=1&env=prodbubble) 
  
 - [ ] ~~Bug1: [Accessibility][Screen Reader]: Narrator in scan mode doesn't announce the role and State of 'Include Persona Details' checkbox.~~ #10850 
- [ ] Bug2: [Accessibility][Visual requirements]: Only  colors are used to show the Persona's status in various sizes. #10851
- [ ] Bug3: [Accessibility][Programmatic access]: Alt of the image under 'Rendering custom coin' is not descriptive. #10853
- [ ] Bug4: [Accessibility][Visual requirements]: All the initial icons of 'Persona' don't have sufficient color contrast. #10854
 
 
 Component: Persona 
@micahgodbolt

**Comment by aneeshack4** [comment]
Split up
